conf/messages.en

```diff
@@ -12,27 +12,17 @@ checkYourAnswers.submit.heading = Send your application
 checkYourAnswers.submit.before.send = Before you send your application, ensure that you have fully reviewed the information you have provided and that it is correct.
 checkYourAnswers.submit.print = You can print a summary of your application on the next page.
 
-site.switchLanguage = Newid yr iaith ir Gymraeg
+site.switchLanguage = Newid yr iaith i’r Gymraeg
 
 date.day = Day
 date.month = Month
 date.year = Year
 
 error.browser.title.prefix = Error:
 error.boolean = Please give an answer
-error.date.day_blank = Enter a day
-error.date.day_invalid = Give a correct day using numbers 1 to 31
-error.date.month_blank = Enter a month
-error.date.month_invalid = Give a correct month using numbers 1 to 12
-error.date.year_blank = Enter a year
-error.date.year_invalid = Give a correct year
 error.required = Please enter a value
 error.summary.title = There is a problem
 
-index.title = Apply for repayment of import duty and import VAT
```

Comment:
AMBER: We have deleted **index.title** and **index.heading** here in the English message file, but have left it in the Welsh message.cy
We have deleted index.guidance from both.  I think we should be consistent with this, so we can match the differences later with confidence.

Reply:
I'll take a look.  I'm going to delay merging this in until after the two review fix changes.  I can then run the "compare" task again and update the file